amclient/utils.py

```diff
@@ -164,3 +186,38 @@ def fsdecode(filename):
             return filename.decode(encoding)
         else:
             raise TypeError("expect bytes or str, not %s" % type(filename).__name__)
+
+
+def package_name_from_path(current_path, remove_uuid_suffix=False):
+    """Return name of package without file extensions from current path.
+    This helper works for all package types (e.g. transfer, AIP, AIC).
+    :param current_path: Current path to package.
+    :param remove_uuid_suffix: Optional boolean to additionally remove
+    UUID suffix.
+    :returns: Package name minus any file extensions.
+    """
+    path = Path(current_path)
+    name, chars_to_remove = path.name, 0
+    if remove_uuid_suffix is True:
+        chars_to_remove = PACKAGE_UUID_SUFFIX_LENGTH
+    for suffix in reversed(path.suffixes):
+        if suffix not in PACKAGE_EXTENSIONS:
+            break
+        chars_to_remove += len(suffix)
+    # Check if we have characters to remove to avoid accidentally
+    # returning an empty string with name[:-0].
+    if not chars_to_remove:
+        return name
+    return name[:-chars_to_remove]
+
+
+def relative_path_to_aip_mets_file(uuid, current_path):
+    """Return relative path to AIP METS file.
+    :param uuid: AIP UUID.
+    :param current_path: Current path to AIP.
+    :returns: Relative path to AIP METS file.
+    """
+    package_name_without_extensions = package_name_from_path(current_path)
+    mets_name = "METS.{}.xml".format(uuid)
+    mets_path = "{}/data/{}".format(package_name_without_extensions, mets_name)
```

Comment:
Nitpick: you could set `mets_path` with a single `format()` instead of assigning `mets_name` first.